ci/build.sh

```diff
@@ -140,6 +140,7 @@ aarch64-fuchsia \
 armv5te-unknown-linux-gnueabi \
 armv5te-unknown-linux-musleabi \
 i686-pc-windows-gnu \
+loongarch64-unknown-linux-gnu \
```

Comment:
This should be removed: it will cause CI to fail since the target hasn't been added to rustc yet.

Reply:
a.  I have removed loongarch64-unknown-linux-gnu in ci/build.rs.
b.  Added loongarch64 support in libc-test and completed the test locally. But for libc-test is it better to submit a PR separately?
See the test result: https://github.com/zhaixiaojuan/libc/commit/467c34ca3d2ebc1c6c72fba8cb155df65390bff1
c. Adjusted the loongarch64/mod.rs file according to the test of libc-test.

Reply:
You should add it in a separate PR once loongarch64-unknown-linux-gnu is available on nightly rust.